smack-extensions/src/main/java/org/jivesoftware/smackx/muc/MultiUserChatManager.java

```diff
@@ -199,6 +206,33 @@ public void processStanza(Stanza packet) {
         };
         connection.addAsyncStanzaListener(invitationPacketListener, INVITATION_FILTER);
 
+        // Listens for all messages that include an XEP-0249 GroupChatInvitation extension and fire the invitation
+        // listeners
+        StanzaListener directInvitationPacketListener = new StanzaListener() {
+            @Override
+            public void processStanza(Stanza packet) {
```

Comment:
We try to phase out the term `packet` in Smack, as it is not XMPP-ish.
```suggestion
            public void processStanza(Stanza stanza) {
```